.github/workflows/continuous-integration.yml

```diff
@@ -59,4 +59,14 @@ jobs:
         run: php data/bin/check_configuration.php
 
       - name: Run Tests
```

Comment:
Would `continue-on-error` work here as well? Maybe with something like 

```yml
- name: Run Tests
  run: ...
  continue-on-error: ${{ matrix.mode == "experimental"}}
```

Reply:
It is possible that it works. Why didn't I find it in the documentation when I searched?

Reply:
I believe the `continue-on-error` will only make sure the other PHP versions still run and not abort as soon as one fails. But if one fails, the CI will stll fail.

Reply:
It's true! The status is also important here to keep the result green.